source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -87,6 +87,42 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter)
             return _acqFileRepository.GetPage(filter, userRegions, personId);
         }
 
+        public List<AcquisitionFileExportModel> GetAcquisitionFileExport(AcquisitionFilter filter)
+        {
+            _logger.LogInformation("Searching all Acquisition Files matching the filter: {filter}", filter);
+            _user.ThrowIfNotAuthorized(Permissions.AcquisitionFileView);
+
+            // Limit search results to user's assigned region(s)
+            var pimsUser = _userRepository.GetUserInfoByKeycloakUserId(_user.GetUserKey());
+            var userRegions = pimsUser.PimsRegionUsers.Select(r => r.RegionCode).ToHashSet();
+            long? personId = pimsUser.IsContractor ? pimsUser.PersonId : null;
+
+            var acqFiles = _acqFileRepository.GetAcquisitionFileExport(filter, userRegions, personId);
+
+            return acqFiles.SelectMany(file => file.PimsPropertyAcquisitionFiles.Where(fp => fp.AcquisitionFileId.Equals(file.AcquisitionFileId)).DefaultIfEmpty(), (file, fp) => (file, fp))
+                                .Select(fileProperty => new AcquisitionFileExportModel
+                                {
+                                    FileNumber = fileProperty.file.FileNumber ?? string.Empty,
+                                    LegacyFileNumber = fileProperty.file.LegacyFileNumber ?? string.Empty,
+                                    FileName = fileProperty.file.FileName ?? string.Empty,
+                                    MotiRegion = fileProperty.file.RegionCodeNavigation?.Description ?? string.Empty,
+                                    MinistryProject = fileProperty.file.Project != null ? $"{fileProperty.file.Project.Code} {fileProperty.file.Project.Description}" : string.Empty,
+                                    CivicAddress = (fileProperty.fp?.Property != null && fileProperty.fp.Property.Address != null) ? fileProperty.fp.Property.Address.FormatFullAddressString() : string.Empty,
+                                    GeneralLocation = (fileProperty.fp?.Property != null) ? fileProperty.fp.Property.GeneralLocation : string.Empty,
+                                    Pid = fileProperty.fp is not null && fileProperty.fp.Property.Pid.HasValue ? fileProperty.fp.Property.Pid.ToString() : string.Empty,
+                                    Pin = fileProperty.fp is not null && fileProperty.fp.Property.Pin.HasValue ? fileProperty.fp.Property.Pin.ToString() : string.Empty,
```

Comment:
**nit:** I see mixed usage of `!= null` and `xyz is not null` - would appreciate sticking to one pattern

Reply:
updated